python/lsst/ctrl/mpexec/cli/cmd/commands.py

```diff
@@ -19,9 +19,11 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import os
+import pathlib
 import sys
 from functools import partial
-from typing import Any
+from typing import Any, Tuple
```

Comment:
We can use python 3.10 in this package so using `tuple` is fine and no need to import from `typing`.